### PR TITLE
Create a user search history function 

### LINKED ISSUE #4: User search history functionality
> AS A user clicking on an empty searchbar
> I WANT to be presented with search history as a drop-down list
> SO THAT I can quickly see something I searched for in the past

### PR DESCRIPTION
**Pull request by leontran44**
Add drop down list and dynamically add value functions

### CODE REVIEW

**Comment by leontran44** [comment]
ah silly me @kevinvongmany, I tested the code and played around with it and commented out a piece of code that clear out the the existing value when adding the a new value in. So the it will not loop again when add new value
code

File: index.html [hunk 1]
```diff
@@ -7,7 +7,13 @@
...
 </head>
 <body>
-    
+    <label for="dropDownListElements">Previous Song searched:</label>
+    <select id="dropDownListElements"></select> 
+    <!--The for attribute in the <label> tag should match the id of the associated form element. 
+    In this case, it should reference the id of the <select> element -->
+    <input type="text" id="newOptionInput" placeholder="New option">
+    <button id="addSongBtn">Add Songs</button>
+    <script src="script.js"></script>
```
Comment (kevinvongmany) [inline index.html]: Could you update line #16 so it's pointing to the new file? Once done, I'll approve this.

**Review by kevinvongmany — CHANGES_REQUESTED** [review]
Just update the source for the script.js and we're good!

**Review by kevinvongmany — APPROVED** [review]

**Review by Cbeattie97 — APPROVED** [review]
Looking good!

**Pull request merged**